### ISSUE TITLE
Improve Okta registration user experience

### ISSUE THREAD

**Issue by grumpyinca** [thegrumpys/odop#341]
New users are getting confused during Okta account creation (registration).   First time registrations both in the email request registration flow and in the Sign Up a.k.a. Self-Service Registration flow are problematic.  These words on the User Accounts page in GitHub Pages (a.k.a. Online Help): 

"_... Do not expect the ODOP:Spring app to appear on Okta’s list of apps and do not be concerned about a “You don’t have any apps.” message on your personal Okta home page (used for adding apps, changing or recovering passwords, etc.). Once you have established a password, simply launch the program in order to get to the sign-in page._"

 are apparently not enough to avoid the problem

Possible solution: [Redirect to Login or Application After Activation](https://devforum.okta.com/t/redirect-to-login-or-application-after-activation/783/24)

**Comment by 1fifoto** [comment]
What exactly do you want? For example, do you want a text change with more steps and detail? If programming, what are you expecting?

**Comment by grumpyinca** [comment]
The "Possible solution" link above is a potential programming solution.  I have not yet invested the time to fully understand the details of the discussion and how much of it would apply to our use of the Okta Sign-in Widget.

I plan to do additional web searches in the hope of learning if it is possible to add the ODOP app to Okta's list for anyone registering.

Also, I will pursue both short term and long term changes to words on the SpringDesignSoftware.org website to better deal with the user confusion issues.

**Comment by grumpyinca** [comment]
The file linked here contains screenshots of both the Self-Service Registration flow and the email request registration flow.

[OktaRegistration.pdf](https://github.com/thegrumpys/odop/files/4248148/OktaRegistration.pdf)

The difficult registration user experience happens only in the email request registration flow.  This reduces the priority of a code fix because users can be warned in advance via email.

**Comment by grumpyinca** [comment]
We made good progress on this issue by changing the Self-Service Registration configuration parameters at:  [Okta admin Self-Service Registration](https://dev-729070-admin.okta.com/admin/self-service-registration)

Note: following the link above will require logging in to Okta as an administrator.

**Assign to group** was changed to "Production Users"

**Default redirect** was changed to "Custom URL" and "https://odop.herokuapp.com/"

This change has addressed the confusing UI for the Self-Service Registration flow but not the email request flow.  Also, there is a tiny blemish in that it leaves the user with a dangling tab.
